docs/products/postgresql/get-started.md

```diff
@@ -11,88 +11,171 @@ Start using Aiven for PostgreSQL® by creating a service, connecting to it, and
 - [psql](https://www.postgresql.org/download/) command line tool installed on your
   computer
 
-## Create a service
+## Set up a service
 
 [Create an Aiven for PostgreSQL® service](/docs/platform/howto/create_new_service)  in the
 [Aiven Console](https://console.aiven.io).
 
-## Connect to the new service
+<details>
+  <summary>Configure your service if the default service setup doesn't meet your needs.</summary>
+  <div>
+    <ol>
+      <li>Select the new service from the list of services on the **Services** page.</li>
+      <li>On the **Overview** page, select **Service settings** from the sidebar.</li>
+      <li>In the **Advanced configuration** section, make changes to the service
+      configuration.</li>
+    </ol>
+    <div>See the available configuration options in [Advanced parameters for Aiven for
+    PostgreSQL](/docs/products/postgresql/reference/advanced-params).</div>
+  </div>
+</details>
+
+## Connect to the service
+
+[Connect to your new service](/docs/products/postgresql/howto/connect-psql) with, for
+example, [psql](https://www.postgresql.org/download/) CLI tool.
 
 :::tip
-Check different tools you can use to connect to Aiven for PostgreSQL in
+Check more tools for connecting to Aiven for PostgreSQL in
 [Connect to Aiven for PostgreSQL](/docs/products/postgresql/howto/list-code-samples).
 :::
 
-[Connect to your new service](/docs/products/postgresql/howto/connect-psql) with, for
-example, [psql](https://www.postgresql.org/download/) CLI tool.
-
 ## Load a test dataset
 
-If you're checking out PostgreSQL, loading a test dataset will give you
-something to look at. This example uses `dellstore2`, a standard store
-dataset with products, orders, inventory and customer information.
+`dellstore2` is a standard store dataset with products, orders, inventory, and customer
+information.
 
 1.  Download the `dellstore2-normal-1.0.tar.gz` file from the
     [PostgreSQL
     website](https://www.postgresql.org/ftp/projects/pgFoundry/dbsamples/dellstore2/dellstore2-normal-1.0/)
     and unzip it.
 
-1.  Navigate to the `dellstore2-normal-1.0` folder on your terminal.
+1.  `cd` to the `dellstore2-normal-1.0` folder using your terminal.
 
-1.  Connect to your PostgreSQL instance with `psql` as shown above.
+1.  [Connect to your PostgreSQL instance with `psql`](/docs/products/postgresql/howto/connect-psql).
 
-1.  Create a `dellstore` database and connect to it with the following
-    command from `psql`:
+1.  Create a `dellstore` database and connect to it:
```

Comment:
Let's not tell devs about `cd` and terminals, that's their bread and butter 🙂 

I'd argue we can just delete 53, 55, and update 57 to 

`From the folder where you extracted the zip, connect to your Postgr...`